README.md

```diff
@@ -13,7 +13,7 @@ Requirements
   strongly recommend that you use a
   [supported version of PHP](https://secure.php.net/supported-versions.php).
 - MySQL 5.0.15 or higher with PDO enabled
-- Apache (recommended) or Nginx web server
+- Apache (recommended) or Nginx webserver
```

Comment:
I agree with the rest of the changes, but not with this one. Please see: 

- https://en.wikipedia.org/wiki/Nginx
  > Nginx, stylized as NGINX, nginx or NginX, is a **_web server_** that...
- https://www.nginx.com
  > NGINX | High Performance Load Balancer, **_Web Server_**
- https://en.wikipedia.org/wiki/Apache_HTTP_Server
  > The Apache HTTP Server, colloquially called Apache, is a free and open-source cross-platform **_web server_** software...

Reply:
Sure! I will change "webserver" back to how it was!

Reply:
@klonos I have changed "webserver" as suggested by you